src/pages/SingleCoursePages.js

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect} from "react";
+import { Routes, Route, Link, NavLink, useNavigate, useParams } from 'react-router-dom'
+import apiUtil from '../utils/apiUtil'
+
+//replcae this with the API call. (Finsished)
+//const course = require('./data/courseData.json') 
+
+//localhost:3000/homepage/course/1
+
+//This function is going to be used to display the single course page from when the
+//user selects the course.  
+function DisplayCoursePage(){
+//using useParams() here to grab the number in the course they select then finding the data
+//associated with that course.
+let { paramID } = useParams()
+
+useEffect(() => {
+    const getCourseData = async () => {
+        const response = await apiUtil('get', `/courses/${paramID}`)
+
+        if (response.status === 200) {
+            return (
+                <div>
+                    <h2>{course[paramID].course_name}</h2>
+                    <p>{course[paramID].crn}</p>
+                    <p>{course[paramID].course_instructor}</p>
+                </div>
+            )
+        }
+
+        else {
```

Comment:
We should probably add else if statements and better error handling than this, perhaps considering error messages that the API returns, and even redirect when someone isn't logged in. The login is something I will work on this week, but the error message is probably something that could be addressed now. We should create a component that serves the purpose of rendering a notification, which maybe we allow it to be an error notification or an info notification with a different color and icon depending on which type it is - I could probably send you a file that I've made for class that we could use